### PR TITLE
Source Snapchat Marketing: Migrate to manifest-only

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What
Migrates source-snapchat-marketing to manifest-only format

## How
Ran the airbyte-ci commands [`migrate-to-manifest-only`, `bump-version`, `format fix`, `pull-request`]

## Review Guide
1. manifest.yaml: should be at the root level of the folder, list version `4.3.0`, contain an inlined `spec`, and all `$parameters` should be resolved.
2. metadata.py: should list the correct `manifest-only` language tag and point to a recent version of `source-declarative-manifest`
3. acceptance-test-config.yml: should correctly point to `manifest.yaml` for the spec test
4. readme.md: Should be updated with correct references to the connector (this has historically been a sticking point with our READMEs)
5. Pretty much everything else should be nuked

## User Impact
None

### CODE REVIEW

**Comment by bazarnov** [comment]
It's incredibly hard to follow the implementations of the source; everything is mixed up for the final `manifest.yaml` file altogether. Can we leave the actual manifest as it was instead of mixing schemas and everything up to the actual stream?

**Comment by girarda** [comment]
@bazarnov I recommend using the connector builder when working on low-code connectors. Working directly with the manifest will indeed be very tedious

File: airbyte-integrations/connectors/source-snapchat-marketing/metadata.yaml [hunk 1]
```diff
@@ -17,9 +17,9 @@ data:
   name: Snapchat Marketing
   remoteRegistries:
     pypi:
-      enabled: true
+      enabled: false
```
Comment (girarda) [inline airbyte-integrations/connectors/source-snapchat-marketing/metadata.yaml:20]: why are we removing the connector from pypi? Is there another path to using it from pyairbyte?
